app/routes/ready-reckoner/select-standard.js

```diff
@@ -1,6 +1,5 @@
-const standards = require('../../services/standards')
+const { setSelectedStandards, getCalculationResult, getCorrelationId, getSelectedStandards, setCalculationResult } = require('./session-handler')
```

Comment:
nit: I might have just done `const sh = require('./session-handler')` to keep this line a bit shorter.